scripts/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+echo -n Start > message.txt
```

Comment:
`serve.sh`と同様に書いた方がいい
```suggestion
echo -n "Start" > message.txt
```